tests/Frontend/CoreSpec.hs

```diff
@@ -277,7 +282,7 @@ instance ArbFormPageResult PageHomeWithLoginPrompt where
 instance ArbFormPageResult CreateTopic where
     arbFormPageResult (CreateTopic space ideas) =
             set protoTopicIdeaSpace space
-          . set protoTopicIdeas ideas
+          . set protoTopicIdeas (map (^. _Id) ideas)
```

Comment:
`(view _Id <$> ideas)`?  I have no strong opinion, but we seem to prefer `view` over operator sections.